plugin/metrics/prometheus/metricsstore/reader_test.go

```diff
@@ -330,6 +331,7 @@ func TestGetRoundTripper(t *testing.T) {
 				TLS: tlscfg.Options{
 					Enabled: tc.tlsEnabled,
 				},
+				AllowTokenFromContext: true,
```

Comment:
@albertteoh I was thinking about this when i was creating test case.

Do we want users to set AllowTokenFromContext config every time when they try to pass bearer token? 
If someone misses out on setting config , and has not enabled token from context , this would give an error. 
So, i was thinking if we should set as [THIS](https://github.com/ChillOrb/jaeger/blob/7b1be1a793200123bb3facd4ce3632d39f2fc0a5/plugin/metrics/prometheus/metricsstore/reader.go#L283) by default

Reply:
I think we should remove `AllowTokenFromContext` and remove [the line of code](https://github.com/ChillOrb/jaeger/blob/7b1be1a793200123bb3facd4ce3632d39f2fc0a5/plugin/metrics/prometheus/metricsstore/reader.go#L283) you pointed to.

That simplifies the `TestGetRoundTripperToken` a lot as we don't need to consider token propagation through context. In fact, we could remove the table driven test cases since there'd only be a single test case remaining.